coralproject/talk
# [CORL-1759] Live Chat: Scroll report dialog into view when shown

## What does this PR do?

Scrolls report dialog into view when shown.

Prevents it from being hidden below the scrolled state of Virtuoso when clicking `Report` on comments near bottom of stream scroll state.

## What changes to the GraphQL/Database Schema does this PR introduce?

None

## How do I test this PR?

- Go to live chat stream
- Scroll to bottom / jump to live
- Click report on bottom most comment
- See that stream scrolls the report dialog into view under the comment
